help/en/html/doc/Technical/gitdeveloper.shtml

```diff
@@ -66,9 +98,13 @@ When you commit, you're saving a copy of your changes into your local
 repository where they're safe, and can eventually be moved onward.
 
   <pre><code>
-  git commit -m&quot;useful comment&quot;
+  git commit
   </code></pre>
 
+When you do this, Git will open an editor window for your commit note. The top line becomes the summary, 
```

Comment:
GitHub recommends the summary be limited to 50 characters or less, so it can be displayed in constrained UIs. Do we want to note that?